script/PresignPauseFromJson.s.sol

```diff
@@ -19,25 +20,38 @@ contract PresignPauseFromJson is MultisigBuilder, JsonTxBuilderBase {
         allowed[1] = _ownerSafe(); // The nonce is updated in the  Foundation Operations Safe (FOS).
     }
 
-    function _addGenericOverrides() internal view virtual override returns (SimulationStateOverride memory override_) {
+    function _simulationOverrides()
```

Comment:
@Ethnical Do you mind sanity checking that the new presigned pause logic in here still works? We'll need to generate new ones soon once we move the DeputyGuardianModule behind a proxy, so just want to validate this to avoid any issues at that time

Reply:
In 5f7cd28 I updated the most recent mainnet presigned pause file accordingly to serve as a template